commands/inspector.go

```diff
@@ -89,7 +89,6 @@ Prints out information about filecoin process and its environment.
 			sw.Printf("\nEnvironment\n")
 			sw.Printf("FilAPI: \t%s\n", info.Environment.FilAPI)
 			sw.Printf("FilPath:\t%s\n", info.Environment.FilPath)
-			sw.Printf("GoPath: \t%s\n", info.Environment.GoPath)
```

Comment:
I am for this change iff we are sure the information is no longer helpful. I think this could still be valuable for certain scenarios, wbu?